server/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/AbstractInternalTerms.java

```diff
@@ -297,14 +297,14 @@ public InternalAggregation get() {
                         reduceContext.consumeBucketsAndMaybeBreak(1);
                         result.add(bucket.reduced(AbstractInternalTerms.this::reduceBucket, reduceContext));
                     } else {
-                        otherDocCount[0] += bucket.getDocCount();
+                        otherDocCount += bucket.getDocCount();
                     }
                 });
             } else if (reduceContext.isFinalReduce()) {
                 TopBucketBuilder<B> top = TopBucketBuilder.build(
                     getRequiredSize(),
                     getOrder(),
-                    removed -> otherDocCount[0] += removed.getDocCount(),
+                    removed -> otherDocCount += removed.getDocCount(),
```

Comment:
I thought you cannot use mutable variables like that in lambda functions.

Reply:
You can if they are a field on an object, just not mutable locals :)

Reply:
👍